ddtrace/contrib/boto/patch.py

```diff
@@ -86,14 +86,15 @@ def patched_query_request(original_func, instance, args, kwargs):
                 span.set_tag(arg[0], arg[1])
 
         # Obtaining region name
-        region = getattr(instance, "region", None)
-        region_name = get_region_name(region)
+        region_name = _get_instance_region_name(instance)
 
         meta = {
             "aws.agent": "boto",
             "aws.operation": operation_name,
-            "aws.region": region_name,
         }
+        if region_name:
+            meta["aws.region"] = region_name
```

Comment:
can you move these as a constants? better in a `constants.py` module. So they're all in one place + you use the same variable in 2 parts of the code